src/main/resources/static/js/thing.js

```diff
@@ -70,58 +70,58 @@ function createThing() {
     $.ajax({
         type: "POST",
         url: "thing/create",
-        datatype: 'json',
-        contentType: 'application/json',
+        datatype: "json",
+        contentType: "application/json",
         data: JSON.stringify(mydata),
         error: function (e) {
             $.notify({
-                message: 'Thing could not be created, check the Log for errors'
+                message: "Thing could not be created, check the Log for errors"
             }, {
                 allow_dismiss: true,
-                type: 'danger',
+                type: "danger",
                 placement: {
                     from: "top",
                     align: "left"
                 },
                 animate: {
-                    enter: 'animated fadeInDown',
-                    exit: 'animated fadeOutUp'
+                    enter: "animated fadeInDown",
+                    exit: "animated fadeOutUp"
                 },
                 z_index: 9000
             });
             addToLog(e.responseText);
         },
         success: function (e) {
             $.notify({
-                message: 'Thing created.'
+                message: "Thing created."
             },{
                 allow_dismiss:true,
-                type: 'info',
+                type: "info",
                 placement: {
                     from: "top",
                     align: "left"
                 },
                 animate: {
-                    enter: 'animated fadeInDown',
-                    exit: 'animated fadeOutUp'
+                    enter: "animated fadeInDown",
+                    exit: "animated fadeOutUp"
                 }
             });
-            addToLog('Thing created');
-            closeModal('dialog');
+            addToLog("Thing created");
+            closeModal("dialog");
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['closeModal' is not defined. (no-undef)](https://app.codacy.com/app/FraunhoferIOSB/ChillImport/pullRequest?prid=4696407)